src/components/Register/Register.tsx

```diff
@@ -1,286 +1,94 @@
-                                                className={
-                                                    this.state.isStudent
-                                                        ? 'btn-special-button btn-special-button1 btn active'
-                                                        : 'btn-special-button btn-special-button1 btn'
-                                                }
-                                                role="button"
-                                                tabIndex="0">
-                                                <User color="black" />
-                                                <span>Student</span>
-                                            </div>
-
-                                            <div
-                                                onClick={(e) => this.handleSpecial(e, 'Teacher')}
-                                                style={{ outline: 'none' }}
-                                                className={
-                                                    this.state.isStudent
-                                                        ? 'btn-special-button btn-special-button2 btn'
-                                                        : 'btn-special-button btn-special-button2 btn active'
-                                                }
-                                                role="button"
-                                                tabIndex="0">
-                                                <span>Teacher</span>
-                                                <TeacherIcon />
-                                            </div>
-                                        </div>
-
-                                        <div className="btn-ui-container">
-                                            <button
-                                                onClick={this.handleClick}
-                                                className="btn-ui btn-weiter"
-                                                type="button">
-                                                Weiter
-                                            </button>
-                                        </div>
-                                    </span>
-                                );
-                            })()}
+                <div className="header">
+                    <span>Register</span>
+                </div>
+                <div className="register-content">
+                    <div className="first-page flex-column-box">
+                        <div className="inputs flex-column-box">
+                            <TextInput
+                                placeholder="E-Mail"
+                                onChange={(value) => registerStore.setEmail(value.target.value)}
+                                className="text-input"
+                                height="2.69rem"
+                                width="28rem"
```

Comment:
looks like every object with `text-input` uses the same height and width. Consider moving them to the css